ztp/kube-compare-reference/required/sriov-operator/SriovOperatorConfig.yaml

```diff
@@ -23,6 +23,12 @@ spec:
   #          openshift.io/<resource_name>:  "1"
   #        requests:
   #          openshift.io/<resource_name>:  "1"
+  {{- if hasKey .spec "enableInjector" }}
```

Comment:
Using `hasKey` will ensure these values are enforced if they're present either in the defaults that create source-crs, or in the CRs we're comparing, but fully ignored if they're not present (as they are optional in the json, and these zero-values will be counted by json marshallers as "unset" and therefore be omitted)